src/zenml/environment.py

```diff
@@ -100,12 +101,12 @@ def in_docker() -> bool:
 
     @staticmethod
     def in_google_colab() -> bool:
-        """If the current python process is running in a Google Colab."""
+        """If the current Python process is running in a Google Colab."""
         return "COLAB_GPU" in os.environ
 
     @staticmethod
     def in_notebook() -> bool:
-        """If the current python process is running in a notebook."""
+        """If the current Python process is running in a notebook."""
         try:
             from IPython import get_ipython  # type: ignore
```

Comment:
Specifically I would say that the behavior of this function is incorrect — it returns true when running in an ipython REPL, and I would expect it to be false.

Reply:
Yep for this that's correct. This was old code prior to my PR which tests for whether it's a notebook in probably not the best way. I'll try to figure out a way to combine these two static methods.